modin/conftest.py

```diff
@@ -693,3 +693,28 @@ def s3_resource(s3_base):
         if not cli.list_buckets()["Buckets"]:
             break
         time.sleep(0.1)
+
+
+@pytest.fixture
+def modify_config(request):
```

Comment:
this fixture was moved from `test_internals.py` in order to use it in `test_groupby.py` as well